schemas/region.schema.json

```
@@ -0,0 +1,133 @@
+  "properties": {
+    "code": {
+      "description": "The unique identifier for a region.",
+      "type": "string"
+    },
+    "timezone": {
+      "description": "A Java valid time zone id.",
+      "type": "string"
+    },
+    "locale": {
+      "description": "Locale information",
+      "type": "object",
+      "properties": {
+        "language": {
+          "description": "An ISO 639 alpha-2 or alpha-3 language code, or a language subtag up to 8 characters in length.",
+          "type": "string"
+        },
+        "country": {
+          "description": "An ISO 3166 alpha-2 country code or a UN M.49 numeric-3 area code.",
+          "type": "string"
+        }
+      },
+      "required": [ "language", "country" ]
+    },
+    "coverage": {
+      "description": "Coverage details.",
+      "type": "object",
+      "properties": {
+        "defaultLocation": {
+          "description": "Default location of this region.",
+          "type": "object",
+          "properties": {
+            "lat": {
+              "description": "Latitude coordinate.",
+              "type": "number",
+              "minimum": -90,
+              "maximum": 90
+            },
+            "lng": {
+              "description": "Longitude coordinate.",
+              "type": "number",
+              "minimum": -180,
+              "maximum": 180
+            },
+            "address": {
+              "description": "City name of this location.",
+              "type": "string"
+            }
+          },
+          "required": [ "lat", "lng", "address" ]
+        },
+        "polygon": {
+
+        }
+      },
+      "required": [ "defaultLocation", "polygon" ]
+    },
+    "currency": {
+      "description": "Details of the currency used in this region.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The ISO 4217 currency code of this currency.",
+          "type": "string"
+        },
+        "symbol": {
+          "description": "The symbol of this currency for the specified locale. For example, for the US Dollar, the symbol is '$'' if the specified locale is the US, while for other locales it may be 'US$'",
+          "type": "string"
+        },
+        "format": {
+          "description": "Format details of the currency.",
+          "type": "object",
+          "properties": {
+            "unit": {
+              "description": "The main unit pattern of this currency. Example: '£0'",
+              "type": "string"
+            },
+            "subunit": {
+              "description": "The subunit pattern of this currency if any. Example: '£0.00'",
+              "type": "string"
+            }
+          },
+          "required": [ "unit" ]
+        }
+      },
+      "required" : [ "format" ]
+    }, 
+    "measurementSystem": {
+      "description": "Measurement system to use in this region.",
+      "enum": ["metric", "imperial"],
+      "default": "metric"
+    }, 
+    "trafficHand": {
```

That's a funny name, how about this instead:

```suggestion
    "trafficRule": {
```